### PR TITLE
Hotfix/seqware 2004 decider fix

### PR DESCRIPTION
**Pull request by SolomonShorser-OICR**

### CODE REVIEW

**Comment by SolomonShorser-OICR** [comment]
ok to test

**Comment by SolomonShorser-OICR** [comment]
ok to test

**Comment by denis-yuen** [comment]
retest this please

File: seqware-pipeline/src/main/java/net/sourceforge/seqware/pipeline/plugins/fileprovenance/ProvenanceUtility.java [hunk 1]
```diff
@@ -122,15 +126,29 @@ public static boolean checkForValidOptions(OptionSet set) {
...
                     for (String value : (List<String>) swaValues) {
                         List<Study> studiesByName = metadata.getStudyByName(value);
-                        for (Study study : studiesByName) {
-                            swaStrings.add(String.valueOf(study.getSwAccession()));
+                        // User must be notified if the sample name or root sample name was invalid (could not be found).
+                        if (studiesByName == null || studiesByName.size() == 0) {
+                            String errorText = "The study with the name \"" + value + "\" could not be found.";
+                            Log.error(errorText);
+                            throw new NotFoundException(errorText);
+                        } else {
```
Comment (denis-yuen) [inline seqware-pipeline/src/main/java/net/sourceforge/seqware/pipeline/plugins/fileprovenance/ProvenanceUtility.java]: This doesn't need to be enclosed in an else clause, same with the next section.

**Comment by denis-yuen** [comment]
Looks good other than that minor annoyance, I appreciate the testing.

**Comment by denis-yuen** [comment]
re-targetting to hotfix branch

**Pull request closed without merging**